### PR TITLE
[BD] Upgrade requirements

### PR DESCRIPTION
**Pull request by morenol**
## Description 
Run make upgrade.

Related to https://openedx.atlassian.net/browse/BOM-1359

Added constraint to readme-renderer

## Reviewers
- [ ] @andrey-canon
- [x] Ready for edX review
- [ ] @jmbowman

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @morenol! I've created [OSPR-4251](https://openedx.atlassian.net/browse/OSPR-4251) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

File: requirements/base.txt [hunk 1]
```diff
@@ -22,16 +22,16 @@ django-crum==0.7.5        # via edx-rbac
...
-django==1.11.29           # via -c requirements/constraints.txt, -r requirements/base.in, django-crum, django-fernet-fields, django-model-utils, edx-django-utils, edx-drf-extensions, edx-rbac, rest-condition
-djangorestframework-jwt==1.11.0  # via edx-drf-extensions
-djangorestframework==3.9.4  # via edx-drf-extensions, rest-condition
+django==1.11.29           # via -c requirements/constraints.txt, -r requirements/base.in, django-crum, django-fernet-fields, django-model-utils, drf-jwt, edx-django-utils, edx-drf-extensions, edx-rbac, rest-condition
+djangorestframework==3.9.4  # via drf-jwt, edx-drf-extensions, rest-condition
 docutils==0.16            # via awscli, botocore
+drf-jwt==1.14.0           # via -c requirements/constraints.txt, edx-drf-extensions
 edx-django-utils==2.0.4   # via -c requirements/constraints.txt, -r requirements/base.in, edx-drf-extensions, edx-rest-api-client
-edx-drf-extensions==4.0.0  # via -r requirements/base.in, edx-rbac
+edx-drf-extensions==4.0.2  # via -r requirements/base.in, edx-rbac
```
Comment (jmbowman) [inline requirements/base.txt]: We should make the Django setting changes mentioned in the 4.0.1 release notes: https://github.com/edx/edx-drf-extensions/releases/tag/4.0.1 .  The changes to `INSTALLED_APPS` mentioned there should also fix compatibility with drf-jwt 0.15.1 .

File: requirements/base.txt [hunk 2]
```diff
@@ -22,16 +22,16 @@ django-crum==0.7.5        # via edx-rbac
 django-fernet-fields==0.6  # via -r requirements/base.in
 django-model-utils==3.2.0  # via -c requirements/constraints.txt, -r requirements/base.in, edx-rbac
 django-waffle==0.18.0     # via -c requirements/constraints.txt, edx-django-utils, edx-drf-extensions
-django==1.11.29           # via -c requirements/constraints.txt, -r requirements/base.in, django-crum, django-fernet-fields, django-model-utils, edx-django-utils, edx-drf-extensions, edx-rbac, rest-condition
-djangorestframework-jwt==1.11.0  # via edx-drf-extensions
-djangorestframework==3.9.4  # via edx-drf-extensions, rest-condition
+django==1.11.29           # via -c requirements/constraints.txt, -r requirements/base.in, django-crum, django-fernet-fields, django-model-utils, drf-jwt, edx-django-utils, edx-drf-extensions, edx-rbac, rest-condition
+djangorestframework==3.9.4  # via drf-jwt, edx-drf-extensions, rest-condition
 docutils==0.16            # via awscli, botocore
+drf-jwt==1.15.1           # via edx-drf-extensions
```
Comment (jmbowman) [inline requirements/base.txt]: Actually, we just discovered today that we can't use drf-jwt>=0.15.0 yet due to a bug in its database migrations under MySQL.  Details in [BOM-1410](https://openedx.atlassian.net/browse/BOM-1410).

Comment (morenol) [reply requirements/base.txt]: I think that there is a typo, it is 1.15.0 instead of 0.15.0, isn't it?

Comment (morenol) [reply requirements/base.txt]: I made a PR in https://github.com/edx/edx-drf-extensions/pull/120 but I think this can be merged as it is right now.

**Review by jmbowman — APPROVED** [review]

**Comment by jmbowman** [comment]
Released: https://pypi.org/project/edx-enterprise-data/1.3.16/

**Pull request merged**